Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -147,6 +149,14 @@ private void OnElectrifiedInteractUsing(EntityUid uid, ElectrifiedComponent elec
             if (!electrified.OnInteractUsing)
                 return;
 
+            // Make an exception for floor tiles, to make placing them around exposed wires less of a hassle.
+            if (HasComp<FloorTileItemComponent>(args.Used))
```

Comment:
Could this not be made generic so it can be applied to more things without extra code? Maybe a PreventUseElectrificationComponent or some better, less clunky name.

Reply:
Well the general component would be the `InsulatedComponent`, which this definitely should be checking for as well. It feels weird to make floor tiles and RCDs "insulated", but I guess that's the correct way to go about it.

On another note, its weird that interacting with wires using your hands doesn't shock you, but using a cheeseburger will. For consistency, `onInteractUsing = true` should probably imply `onHandInteract = true`, but I guess that should be discussed separately.